src/components/DayPicker.jsx

```diff
@@ -498,12 +503,15 @@ export default class DayPicker extends React.Component {
   }
 
   initializeDayPickerWidth() {
-    this.dayPickerWidth = calculateDimension(
-      // eslint-disable-next-line react/no-find-dom-node
-      ReactDOM.findDOMNode(this.calendarMonthGrid).querySelector('.CalendarMonth'),
-      'width',
-      true,
-    );
+    // eslint-disable-next-line react/no-find-dom-node
+    const calendarMonthGridDOMNode = ReactDOM.findDOMNode(this.calendarMonthGrid);
+    if (calendarMonthGridDOMNode) {
```

Comment:
i'd actually wrap the findDOMNode in a truthy check for the ref; if it's nullary, there's no point in doing the findDOMNode